lib/util.js

```diff
@@ -74,7 +74,7 @@ const estimateGas = (web3, opts) => new Promise((resolve, reject) => {
 
 // / Requires a case sensitive name of the contract and will return the ABI if found.
 const getABI = (name) => {
-  return require(`${__dirname}/build/abi/${name}.json`)
+  return require(`${__dirname}/build/contracts/${name}.json`).abi
```

Comment:
Please revert that change and use `abi` files. The reason for using that extra step is to reduce the size of eac.js-lib size, also lib doesn't require the full content of contracts files. Use `extractABI.js` to generate the abi files.